internal/pipe/docker/docker_test.go

```diff
@@ -1101,8 +1098,9 @@ func TestRunPipeWhileUsingBuildpacks(t *testing.T) {
 				Dist:        dist,
 				Dockers:     docker.dockers,
 			})
+
 			ctx.Parallelism = 1
-			ctx.Env = docker.env
+			// ctx.Env = docker.env
```

Comment:
seems like this was not intentional...

Reply:
this was intentional because we pass the environment variables of the context to the environment variables of the command, so, once we do not specify any environment variable for the test, it causes environment variables of command to be empty, so, this is why we get an error like `$HOME is not defined, etc.